### PR TITLE
Make `testUtils` a namespace

### LINKED ISSUE #8: Can it be `namespace testUtils`?
I doubt there's any requirement for `testUtils` be a class instance, so can we make it a namespace instead?

### PR DESCRIPTION
**Pull request by saschanaz**
Fixes #8
Fixes #4

- [x] At least two implementers are interested (and none opposed): Well, Gecko is the only implementer.

***
<a href="https://whatpr.org/testutils/9.html" title="Last updated on Jan 18, 2022, 7:41 PM UTC (c64b61a)">Preview</a> | <a href="https://whatpr.org/testutils/9/61e6887...c64b61a.html" title="Last updated on Jan 18, 2022, 7:41 PM UTC (c64b61a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -68,6 +64,10 @@ The `gc()` method must run these steps:
 
 <h2 class=no-num id=acknowledgements>Acknowledgments</h2>
 
+Thanks to
+Kagami Sascha Rosylight
+for being awesome!
```
Comment (jgraham) [inline index.bs:69]:
```suggestion
Thanks to the following people who have contributed to the
Test Utils standard:

Kagami Sascha Rosylight

You are awesome!
```

File: index.bs [hunk 2]
```diff
@@ -40,21 +40,17 @@ the default shipping configuration of user agents. They must only be
 enabled in testing configurations for example with special build
 flags, or when a specific non-default preference is set.
 
-The TestUtils Object {#the-testutils-object}
+The TestUtils namespace {#the-testutils-object}
```
Comment (jgraham) [inline index.bs:43]: Change the link name, and use capital N for namespace.

**Review by jgraham — COMMENTED** [review]
I think we can try this approach if you're happy to update the gecko patch and existing usage.

File: index.bs (hunk 2 above)
Comment (saschanaz) [reply index.bs:43]:
```suggestion
The TestUtils Namespace {#the-testutils-namespace}
```

Isn't the point of explicit link to keep it same regardless of the title change? Anyway, doing so...

**Review by jgraham — APPROVED** [review]
Thanks!

**Comment by saschanaz** [comment]
> I think we can try this approach if you're happy to update the gecko patch and existing usage.

That shouldn't be hard 👍

File: index.bs (hunk 2 above)
Comment (jgraham) [reply index.bs]: Yeah, but in practice I don't think there are many existing links, and a small amount of breakage now is probably better than having a misleading fragment id for all time.

**Pull request merged**